lib/normalizer.c

```diff
@@ -3488,6 +3488,26 @@ grn_nfkc_normalize_unify_iteration_mark(grn_ctx *ctx,
     return unified_buffer;
   }
 #  undef N_KATAKANA_BYTES
+  else if (current_length == 3 && current[0] == 0xe3 && current[1] == 0x80 &&
+           current[2] == 0xbb && previous_length >= 3 &&
+           GRN_CHAR_TYPE(grn_nfkc_char_type(previous)) == GRN_CHAR_KANJI) {
+    /**
+     * U+303B VERTICAL IDEOGRAPHIC ITERATION MARK
+     * For kanji iteration mark, we simply repeat the previous kanji character.
+     * This implementation only handles simple cases as follows.
+     * - U+5404 U+303B -> U+5404 U+5404
+     * - CJK Unified Ideograph-2000B U+303B ->
+     *     CJK Unified Ideograph-2000B CJK Unified Ideograph-2000B
```

Comment:
Could you mention both of code point ("U+XXXX") and character name?

```suggestion
     * - U+2000B U+303B -> U+2000B U+2000B
     *   ("CJK UNIFIED IDEOGRAPH-2000B" "VERTICAL IDEOGRAPHIC ITERATION MARK" ->
     *    "CJK UNIFIED IDEOGRAPH-2000B" "CJK UNIFIED IDEOGRAPH-2000B")
```